tests/TestCase/Error/DebuggerTest.php

```diff
@@ -825,4 +825,24 @@ public function testPrintVar()
         $expected = sprintf($expected, str_replace(CAKE_CORE_INCLUDE_PATH, '', __FILE__), __LINE__ - 9);
         $this->assertEquals($expected, $result);
     }
+
+    /**
+     * test formatHtmlMessage
+     *
+     * @return void
+     */
+    public function formatHtmlMessage()
```

Comment:
Test prefix missing
Tests are not running

Reply:
Whoops :blush: